vanuatuvoices/util.py

```diff
@@ -11,3 +12,18 @@ def language_detail_html(context=None, request=None, **kw):
     res['consonants_html'], res['consonants_css'] = d.render()
     res['uncovered'] = [p for i, p in enumerate(context.inventory) if i not in covered]
     return res
+
+
+def localize_url(req, lang):
```

Comment:
Maybe this should be moved to `clld.web.util.helpers` eventually.

Reply:
ah, yes, `req.query_params` is cleaner -- and yes, I also think the function is better off in `clld.web.util.helpers`, as it could be reused for other clld applications. -- I'll prepare a pull request.